website/blog/2024-04-30-release-1.10.md

```diff
@@ -0,0 +1,133 @@
+tags: [podman-desktop, release, podman]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.10/1000000-downloads.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.10 Release! 🎉
+
+![Podman-desktop-1-10-hero](img/podman-desktop-release-1.10/1000000-downloads.png)
+
+This release introduces:
+
+- **1 Million Downloads!**: Wow, we made it!
+- **Extension Catalog**: Redesigned extensions page and catalog to get the most out of Podman Desktop.
+- **Podman 5**: [Podman 5.0.2](https://github.com/containers/podman/releases/tag/v5.0.2) now recommended for all users.
+- **Extension API Improvements**: Additional updates to the extension API used by 🦭 Podman Desktop's extensions.
+
+Podman Desktop 1.10 is now available. [Click here to download it](/downloads)!
+
+<!--truncate-->
+
+---
+
+## Release Details
+
+### 1 Million Downloads! 🎉
+
+We've hit over 1 million downloads of Podman Desktop since we started! A big thank you 🙏 to everyone
+who has helped get us to this number - especially those who have opened issues or PRs and contributed
+to this success!
+
+Looking forward to the next million! 🚀
+
+### Extension Catalog
+
+To make it easier to access extensions we added an item to the main navigation and completely revamped
+the Extensions page. You can now see and control all your Podman Desktop and Docker Desktop extensions
+in a single place. If you want to view more details, click on an extension to see the expanded readme.
+
+![extensions](img/podman-desktop-release-1.10/extensions.png)
+
+With the growing number of extensions available we've also introduced a catalog to make it easy to find
+and install them. Check the catalog regularly as we hear there are some exciting new extensions sailing in!
+
+![extension catalog](img/podman-desktop-release-1.10/extension-catalog.png)
+
+### Podman v5 (5.0.2)
+
+Now that we've had some experience and time to test our migration tools, we're glad to make Podman v5 a
+recommended upgrade for all users. Podman Desktop will help ensure you don't lose any data during the
+migration. In particular macOS users will see a performance boost as we've switched to the native
+Apple Hypervisor.
+
+Find out more details in the blog announcement for 🦭 Podman version 5 [here](https://blog.podman.io/2024/03/podman-5-0-has-been-released/).
+
+### Extension API Improvements
+
+We've added support for manifests (multi-arch images) to the extension API to give upcoming extensions
+more capabilites and even better integration into 🦭 Podman Desktop:
+
+- feat: add inspectManifest API endpoint [#6812](https://github.com/containers/podman-desktop/pull/6812)
+- feat: add createManifest API [#6630](https://github.com/containers/podman-desktop/pull/6630)
+
+---
+
+## Other Notable Enhancements
+
+We've added lots of features this release, here are some other highlights:
+
+- feat: add multi-arch building / manifest creation [#6792](https://github.com/containers/podman-desktop/pull/6792)
```

Comment:
don't we think this is worth giving a dedicated paragraph?

Reply:
Yes, we should. Multi-platform section and image added in new commit.